src/apps/relay/mainrelay.c

```diff
@@ -3286,13 +3288,12 @@ int THREAD_cleanup(void) {
   mutex_buf_initialized = 0;
   return 1;
 }
+/* OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_1_0 */
+/* defined(OPENSSL_THREADS) end */
```

Comment:
The #if-checks were combined into a single check.

Maybe these two lines of comments should be replaced with

/* #if defined(OPENSSL_THREADS) && (OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_1_0) */

for clarity?

Reply:
Issue resolved . use  ` -DOPENSSL_THREADS`
```shell
export CFLAGS="${CFLAGS} -DOPENSSL_THREADS  "
```